src/app/shared/components/user-card/user-card.component.ts

```diff
@@ -25,11 +25,12 @@ export class UserCardComponent implements OnInit, OnChanges {
   }
 
   ngOnChanges() {
+    // XXX: When is this called??
     this.fetchLearningObjects();
   }
 
   async fetchLearningObjects() {
-    this.objects = await this.learningObjectService.getUsersLearningObjects(this.user.username);
+    this.objects = await this.userService.getUsersLearningObjects(this.user.username);
```

Comment:
This is the route that should be used for this.

This is from clark-service, it should be in the /learning-objects/search/search.service.ts

```typescript
router.get(
            "/users/:username/learning-objects",
            this.getUsersLearningObjects,
        );
```